doc/about/update_notifications/index.md

```diff
@@ -0,0 +1,53 @@
+# Update Notifications {#about_update_notifications}
+
+[TOC]
+
+Outdated software is a huge issue in scientific workflows. Especially when the software is not part of the packaging
+provided by the operating system, it tends to be installed once and never updated.
+
+We provide update notifications as a convenient service to app users so that they immediately see when a new version is
+available. When we infer that an application is being run by its developer, we additionally remind them to keep their
+checkout of the Sharg library up-to-date. This ensures that new features arrive at app developers faster, applications have fewer bugs, and we receive fewer bug reports on outdated versions.
+
+## How it works
+
+Upon starting an application using the `sharg::argument_parser`, it is determined whether the app was built in release
+mode (we assume a user is running it) or in debug mode (we assume the developer is running it).
+
+We then send a web-request to our server with the following information:
+
+ * The application name
+ * The application version
+ * The Sharg version that the application was built with
+ * The operating system type (Linux, macOS, Windows, or BSD)
+ * The CPU type (32 or 64bit)
+
+However, we send at most one request per day [we keep track of this by saving a timestamp to `~/.config/seqan/app_name`].
+
+We inform the user about available updates, if a newer app version is registered in our database (or can be automatically determined).
+We inform the developer about new Sharg versions, if the app is running in debug mode.
+Additionally, we recommend registering your app with us (support@seqan.de).
+
+## Privacy Implications
+
+Only the information listed above is transmitted; no data is gathered about the usage of the app, and, of course, no
+information regarding your input/output files is transmitted. All our code is open source, and hence this can be verified at any time.
```

Comment:
```suggestion
information regarding your input/output files is transmitted. All our code is open source, and hence this can be
verified at any time.
```